style.scss

```diff
@@ -414,7 +414,7 @@ body >footer {
 					color: $purple-highlight;
 			}
 				
-			&.twitter::before { content: "\e908"; }
+			&.twitter::before { content: "\e909"; }
```

Comment:
FWIW it looks like this action re-numbered things, i.e what used to be 908 is not 909. That's typically considered bad form, since it means the website will look wrong until people get the new version of the font.

Would it be possibly possible to modify the pull request so that no re-numbering is needed?